src/app/state/importaccounts/saga.ts

```diff
@@ -14,24 +14,58 @@ import type Transport from '@ledgerhq/hw-transport'
 import {
   selectImportAccountHasMissingBalances,
   selectImportAccounts,
-  selectImportAccountsOnCurrentPage,
   selectImportAccountsFullList,
+  selectImportAccountsOnCurrentPage,
   selectImportAccountsPageNumber,
+  selectSelectedBleDevice,
 } from './selectors'
 import { getAccountBalanceWithFallback } from '../../lib/getAccountBalanceWithFallback'
+import BleTransport from '@oasisprotocol/ionic-ledger-hw-transport-ble/lib'
+import { ScanResult } from '@capacitor-community/bluetooth-le'
 
 function* setStep(step: ImportAccountsStep) {
   yield* put(importAccountsActions.setStep(step))
 }
 
+function* isBluetoothSupported() {
```

Comment:
A bit more dirty fix, as the underlying library always return true if on web. I am checking if `navigator?.bluetooth?.requestLEScan` is defined. Any idea for cleaner check? 🤔

Reply:
`navigator?.bluetooth?.requestLEScan` seems fine

But I wouldn't add it into canAccessBle:

> ```ts
>    // Scan depends on requestLEScan method, which is not available on the web(feature flag)
>    const isSupportedOnDesktop = !!navigator?.bluetooth?.requestLEScan
> ```

Since requestLEScan is implementation detail of @oasisprotocol/ionic-ledger-hw-transport-ble, it should be part of it. Can it be part of `BleTransport.isEnabled` even if it is web-specific?

Reply:
It also needs to be web-specific. This currently disables bluetooth in Android